ar-001/auth.json

```diff
@@ -0,0 +1,283 @@
+            "confirmation_code_input_error_unknown_error": "خطأ في المصادقة",
+            "continue_button": "تحقق",
+            "didnt_receive_a_code_description": "لم تستلم رمز ؟",
+            "didnt_receive_a_code_link": "إعادة إرسال الرمز ",
+            "choose_another_method_link": "إختر طريقة أخرى"
+        },
+        "verification_code_sms": {
+            "description": "\"${generated code} هو الرمز الذي تستخدمه لمرة واحدة لتسجيل الدخول إلى ${business_name} @ ${receiver_domain} # ${generated code]\""
+        },
+        "choose_organization_page": {
+            "page_title": "اختر مؤسسة |  ${business_name}",
+            "heading": "أين تريد تسجيل الدخول ؟",
+            "choose_org_select_label": "سجل دخولي إلى",
+            "choose_org_select_error_invalid": "مؤسسة غير صحيحة",
+            "continue_button": "المتابعة"
+        },
+        "request_access_page": {
+            "page_title": "طلب اللجوء",
+            "heading": "كن على علم أولا",
+            "description": "سنبقيك على اطلاع دائم ونرسل لك دعوة عندما يكون اللجوء متاحًا",
+            "continue_button": "طلب اللجوء",
+            "success_heading": "كل شيء جاهز",
+            "first_name_input_label": "الإسم الأول",
+            "first_name_input_error_required": "الرجاء إدخال إسمك الأول",
+            "last_name_input_label": "إسم العائلة ",
+            "email_input_label": "بريد الإلكتروني",
+            "email_input_error_invalid": "أدخل بريد إلكتروني صحيح",
+            "email_input_error_required": "مطلوب بريد إلكتروني",
+            "email_input_error_duplicate_email_found": "عنوان البريد الإلكترونى هذا مسجل بالفعل",
+            "success_description": "شكرا لطلب اللجوء.  لقد أضفناك إلى قائمتنا وسنرسل التحديثات قريبًا.",
+            "success_link_url": "العودة إلى موقع ${business_name}",
+            "disclaimer": "من خلال المتابعة، فإنك توافق على سياساتنا",
+            "disclaimer_terms_of_use_link": "الشروط",
+            "disclaimer_privacy_link": "سياسة الخصوصية"
+        },
+        "subscribe_form_widget": {
+            "page_title": "إشترك",
+            "heading": "البقاء في الحلقة",
+            "description": "تلقى منا معلومات حول تحديثات المنتج والأخبار والأحداث.",
+            "name_input_label": "الإسم الأول ",
+            "name_input_error_required": "الرجاء إدخال اسم",
+            "name_input_error_exceeds_max_length": "الاسم طويل جدا",
+            "email_input_label": "بريد إلكتروني",
+            "email_input_error_invalid": "أدخل بريد إلكتروني صحيح",
+            "email_input_error_required": "مطلوب بريد إلكتروني",
+            "email_input_error_duplicate_email_found": "هذا البريد الإلكتروني مشترك بالفعل",
+            "continue_button": "حساب جديد",
+            "success_heading": "تم بنجاح! مرحبا بك في الحلقة.",
+            "credit_powered_by": "مدعوم من طرف"
+        },
+        "recovery_code_verify_page": {
+            "page_title": "إستخدم رمز الاسترداد |  ${business_name}",
```

Comment:
@onderay there looks like additional whitespace here on the right hand side of ` | `